src/amd/compiler/aco_opcodes.py

```diff
@@ -599,12 +617,12 @@ def opcode(name, opcode_gfx9 = -1, format = Format.PSEUDO, input_mod = False, ou
    (0x1f, 0x1f, 0x16, 0x16, 0x1f, "v_mac_f32", True),
    (0x20, 0x20, 0x17, 0x17, 0x20, "v_madmk_f32", False),
    (0x21, 0x21, 0x18, 0x18, 0x21, "v_madak_f32", False),
-   (0x25, 0x25, 0x19, 0x19, 0x25, "v_add_co_u32", False),
```

Comment:
maybe we should add v_add_co_u32_e64 and friends to VOP3 opcodes for RDNA because we need them for 64bit integer calculation.

Reply:
Agreed.